PhoneNumberKit/MetadataManager.swift

```diff
@@ -9,19 +9,19 @@
 import Foundation
 
 final class MetadataManager {
-    
+
     var territories = [MetadataTerritory]()
     var territoriesByCode = [UInt64: [MetadataTerritory]]()
     var mainTerritoryByCode = [UInt64: MetadataTerritory]()
     var territoriesByCountry = [String: MetadataTerritory]()
-    
+
     // MARK: Lifecycle
 
     /**
      Private init populates metadata territories and the two hashed dictionaries for faster lookup.
      */
-    public init () {
-        territories = populateTerritories()
+     public init (metadataPath: String? = nil) {
```

Comment:
I recommend that the metadataPath is a file URL not a String. A String can be anything. You can validate that the URL is a fileURL. The code doesn't change much but using a URL here would make the intention more clear and probably reduce some user bugs.